MekHQ/src/mekhq/service/MassRepairService.java

```diff
@@ -274,46 +438,41 @@ private static int performUnitMassTechAction(CampaignGUI campaignGUI, Unit unit,
 			}
 
 			if (refreshParts) {
-				parts = campaignGUI.getCampaign().getPartsNeedingServiceFor(unit.getId());
+				parts = campaignGUI.getCampaign().getPartsNeedingServiceFor(unit.getId(), true);
 			}
 		}
 
 		if (unit.getEntity().isOmni() && !unit.isSalvage()) {
 			for (PodSpace ps : unit.getPodSpace()) {
 				ps.setRepairInPlace(!configuredOptions.isReplacePodParts());
 			}
-			
-			/*
-				// This needs to be looked at by Neoancient since he put this in. 
-				// I don't think this accomplishes the desire effect.
-			
+
+			// This needs to be looked at by Neoancient since he put this in.
+			// I don't think this accomplishes the desire effect.
+
 			// If we're replacing damaged parts, we want to remove any that have
 			// an available replacement
 			// from the list since the pod space repair will cover it.
 
-			List<IPartWork> temp = new ArrayList<>();
-			
-			for (IPartWork p : parts) {
-				if ((p instanceof Part) && ((Part)p).isOmniPodded()) {					
-					MissingPart m = ((Part) p).getMissingPart();
-					
-					if (m != null && m.isReplacementAvailable()) {
-						continue;
-					}
-				}
-				
-				temp.add(p);
-			}
-			
-			parts = temp;
-			*/
+			// List<IPartWork> temp = new ArrayList<>();
+			//
+			// for (IPartWork p : parts) {
+			// if ((p instanceof Part) && ((Part) p).isOmniPodded()) {
+			// MissingPart m = ((Part) p).getMissingPart();
+			//
+			// if (m != null && m.isReplacementAvailable()) {
+			// continue;
+			// }
+			// }
+			//
+			// temp.add(p);
+			// }
+			//
+			// parts = temp;
```

Comment:
If we're no longer using this code, we should probably just remove it and allow the repository's history track what used to be here.